tests/test_project_admin/test_suite_settings.ts

```diff
@@ -395,4 +395,127 @@ describe('Field binding tests', () => {
 
         expect(emitted(wrapper, 'field_change')[0][0]).toEqual(wrapper.vm.d_suite);
     });
+
+    test('Adding instructor file using batch select', async () => {
+        expect(wrapper.vm.d_suite!.instructor_files_needed.length).toEqual(2);
+
+        wrapper.findAll('.batch-select-button').at(0).trigger('click');
+        await wrapper.vm.$nextTick();
+
+        wrapper.findAll('.batch-select-card').at(2).trigger('click');
+        await wrapper.vm.$nextTick();
+
+        wrapper.find('.modal-confirm-button').trigger('click');
+        await wrapper.vm.$nextTick();
+
+        expect(wrapper.vm.d_suite!.instructor_files_needed.length).toEqual(3);
+    });
```

Comment:
Note that I have not refactored the tests as I wasn't entirely sure that my approach was what @james-perretta had in mind. I didn't want to do anything prematurely.

Reply:
I mostly left these tests here to verify that the props are being passed in properly.